build.gradle

```diff
@@ -28,11 +28,11 @@ dependencies {
     }
     compileOnly 'io.camunda:zeebe-exporter-api:8.1.1'
     compileOnly 'io.camunda:zeebe-elasticsearch-exporter:8.1.1'
-    compileOnly 'org.json:json:20190722'
+    implementation ('org.json:json:20190722')
 }
 
 group = 'org.mifos.phee'
-version = '1.1.1-SNAPSHOT'
+version = '3.0.0-SNAPSHOT'
```

Comment:
increase minor version as backward compatibility is not broken

Reply:
Updated the version to 1.2.0